src/garage/tf/algos/off_policy_rl_algorithm.py

```diff
@@ -13,7 +13,7 @@ class OffPolicyRLAlgorithm(RLAlgorithm):
         replay_buffer (garage.replay_buffer.ReplayBuffer): Replay buffer.
         use_target (bool): Whether to use target.
         discount(float): Discount factor for the cumulative return.
-        n_epoch_cycles (int): Epoch cycles.
+        n_epoch_cycles (int): Number of policies sampled in one epoch.
```

Comment:
Yeah, that explanation isn't any clearer, since the policy is usually the same each time.
A better explanation is just "number of train_once calls per epoch."